common/src/main/java/io/netty/util/ReferenceCountUtil.java

```diff
@@ -133,7 +133,10 @@ public static void safeRelease(Object msg, int decrement) {
      * Schedules the specified object to be released when the caller thread terminates. Note that this operation is
      * intended to simplify reference counting of ephemeral objects during unit tests. Do not use it beyond the
      * intended use case.
+     *
+     * @deprecated as this may introduce a lot of memory usage better explicit release objects.
```

Comment:
nit: `better explicit release objects.` -> ` so it is generally preferable to manually release objects.`